features/step_definitions/inspection_steps.rb

```diff
@@ -43,14 +43,14 @@
     count = 0
     while count < attempts
       break if element_exists("* {text CONTAINS[c] '#{text}'}")
-      scroll_down
+      pan_down
```

Comment:
Ugh, I remember reading the calabash developers say on the Wiki that scroll was the "long term supported way" of accomplishing this but in classic calabash style I can't find any of the resources anymore. 

Seems fine to change it, and if the tests fail we'll know there's an issue.